### PR TITLE
Clarifications to README.md

### PR DESCRIPTION
**Pull request by worldofchris**
This project is a great piece of work which I can see us getting a lot of value out of.

I've proposed a couple of changes to the README which you may see as NOOB overkill but if not I hope they are of value you to you.

As I say, great stuff and thanks to everyone who has contributed to this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Requirements:
 Settings:
 
 * `UPLOAD_DIR` : The directory to store the uploaded file in, within `MEDIA_ROOT`. Defaults to "uploads".
-* `BUFFER_SIZE`: The size of each chunk to write. Defaults to 10 MB.  See the caveat at the bottom before changing it.
+* `BUFFER_SIZE`: The size of each chunk to write. Defaults to 10 MB - i.e. `BUFFER_SIZE = 10485760`.  See the caveat at the bottom before changing it.
```
Comment (skoczen) [inline README.md]: Like the other changes, but I'm not sure this line adds clarity.

If you're trying to say that it's in bytes, I'd add that into the phrasing, or into the note down below - whatever you think works best!

**Comment by worldofchris** [comment]
I'm now using django-ajax-uploader with Fine Uploader, as valum's file-uploader is now called, instead of the version bundled with django-ajax-uploader.  To use this you need to specify a Custom Header when you configure the file uploader instead of using the django-ajax-uploader specific params.

The advantage of this is you can use Fine Uploader out of the box and so take advantage of new features as they are added to it, for example the progress bar.

I have proposed a comment to add to the README.md explaining how to use django-ajax-uploader with Fine Uploader.

In fact, given Fine Uploader now allows you to specify Custom Headers is it worth now removing the bundled version of valum's uploader from django-ajax-uploader?

**Comment by skoczen** [comment]
Hey Chris,

Very interesting, and thanks for the pull request - I'll check out Fine Uploader and this pull shortly!

**Comment by skoczen** [comment]
In better late than never news, finally getting this in. I just added this + credit to you in the readme, instead of dealing with the merge conflicts - will go out as 0.3.5 this afternoon.

Thanks for the improvements!

**Pull request closed without merging**